src/main/webapp/jsp/instructorCourseEnrollResult.jsp

```diff
@@ -17,7 +17,7 @@
   <%=Const.ParamsNames.STUDENTS_ENROLLMENT_INFO%>
 </c:set>
 
-<ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Enrollment Results for ${data.courseId}" jsIncludes="${jsIncludes}">
+<ti:instructorPage pageTitle="Enrollment Results" bodyTitle="Enrollment Results for ${data.courseId}" jsIncludes="${jsIncludes}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the title would include the course ID.